forge-gui/res/cardsfolder/upcoming/parting_gust.txt

```diff
@@ -0,0 +1,10 @@
+Name:Parting Gust
+ManaCost:1 W
+Types:Sorcery
+K:Gift
+SVar:GiftAbility:DB$ Token | TokenAmount$ 1 | TokenScript$ u_1_1_fish | TokenTapped$ True | TokenOwner$ Promised | LockTokenScript$ True | GiftDescription$ a tapped Fish
+A:SP$ ChangeZone | ValidTgts$ Creature.nonToken | Origin$ Battlefield | Destination$ Exile | TgtPrompt$ Select target nontoken creature | RememberTargets$ True | SubAbility$ DelTrig | SpellDescription$ Exile target nontoken creature. If the gift wasn't promised, return that creature to the battlefield under its owner's control with a +1/+1 counter on it at the beginning of the next end step.

```

Comment:
where are you taking RememberTargets from? that's usually not working since it's the wrong object

Reply:
I think I took that from Flicker.